app/views/application/_post.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="card my-4">
+    <div class="card-header-component">
+        <a href="/users/<%=user.id%>/posts/<%=post.id%>"><h4># <%= post.Title%></h4></a>
```

Comment:
- [ ] Kindly fix this error. We should have `title`, not `Title`

  ![image](https://github.com/Izebiafe/solo-blog/assets/81468675/626514d9-7793-429a-bb40-1d7e081889d6)